src/main/java/com/volasoftware/tinder/services/UserService.java

```diff
@@ -23,20 +29,39 @@
 public class UserService {
 
     private final UserRepository userRepository;
-    private final TokenRepository tokenRepository;
-    @Autowired
-    private EmailSenderService senderService;
+    private final VerificationRepository verificationRepository;
+    private final ResourceLoader resourceLoader;
+    private JavaMailSender mailSender;
 
-    public UserService(UserRepository userRepository, TokenRepository tokenRepository) {
+    public UserService(UserRepository userRepository,
+                       VerificationRepository verificationRepository,
+                       ResourceLoader resourceLoader,
+                       JavaMailSender mailSender) {
         this.userRepository = userRepository;
-        this.tokenRepository = tokenRepository;
+        this.verificationRepository = verificationRepository;
+        this.resourceLoader = resourceLoader;
+        this.mailSender = mailSender;
     }
 
     public List<User> getAll() {
         return userRepository.findAll();
     }
 
+    private String getEmailContent(String token) throws IOException{
+        Resource emailResource = resourceLoader.getResource("classpath:emailResources/ConfirmationPage.html");
+
+        File emailFile = emailResource.getFile();
+        Path path = Path.of(emailFile.getPath());
+        String emailContent = Files.readString(path);
+
+        return  emailContent.replace("{{token}}" , "http://localhost:8080/verify/" + token);
```

Comment:
`localhost:8080` should be configurable via application.properties, because is env dependent

Reply:
The application.properties file will be reconfigured in the next task.